config/helm/kustomization.yaml

```diff
@@ -1,100 +1,116 @@
-namespace: '{{ .Release.Namespace }}'
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 
 resources:
-  - input/infrastructure-components.yaml
-  - ssh-sso-public-key-secret.yaml
-  - secret-bootstrap-credentials.yaml
+# Cluster API Provider Azure components
+- bases/infrastructure-components.yaml
+# Secrets
+- bases/secrets/capz-manager-bootstrap-credentials.yaml
+- bases/secrets/cluster-api-provider-azure-ssh-sso-pub-key.yaml
 
-transformers:
-  - common-labels.yaml
-  - webhook-prefix.yaml
+namespace: "{{ .Release.Namespace }}"
+
+labels:
+# Common labels
+- pairs:
+    helm.sh/chart: cluster-api-provider-azure
+    app.kubernetes.io/name: cluster-api-provider-azure
+    app.kubernetes.io/instance: "{{ .Release.Name }}"
+    app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
+    app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+    application.giantswarm.io/branch: "{{ .Values.project.branch }}"
+    application.giantswarm.io/commit: "{{ .Values.project.commit }}"
+    application.giantswarm.io/team: "{{ index .Chart.Annotations \"application.giantswarm.io/team\" }}"
```

Comment:
Do we need any of these as default selector labels?

Reply:
Actually I didn't check that and mostly focused on not changing the contents of `helm/` if possible. As in: Running `make generate` should result in the same chart with or without my change as much as possible.

Reply:
Sounds good!